### PR TITLE
fix(curriculum): Remove unnecessary assert message argument from English Coding Interview Prep challenges - Project Euler - 04

### PR DESCRIPTION
**Pull request by RandellDawson**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Related to issue #36400

This PR removes assert message arguments from `testString` for another 75 `Project Euler` challenges in the `Coding Interview Prep` curriculum section.

### CODE REVIEW

**Review by vkWeb — APPROVED** [review]
@RandellDawson Good job here too. Everything looks fine.

File: curriculum/challenges/english/08-coding-interview-prep/project-euler/problem-34-digit-factorials.english.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Note: as 1! = 1 and 2! = 2 are not sums they are not included.
 ```yml
 tests:
   - text: '<code>digitFactorial()</code> should return { sum: 40730, numbers: [145, 40585] }.'
```
Comment (moT01) [inline curriculum/challenges/english/08-coding-interview-prep/project-euler/problem-34-digit-factorials.english.md]:
```suggestion
  - text: <code>digitFactorial()</code> should return { sum: 40730, numbers: [145, 40585] }.
```

File: curriculum/challenges/english/08-coding-interview-prep/project-euler/problem-34-digit-factorials.english.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ Note: as 1! = 1 and 2! = 2 are not sums they are not included.
...
-    testString: "assert.deepEqual(digitFactorial(), { sum: 40730, numbers: [145, 40585] }, '<code>digitFactorial()</code> should return { sum: 40730, numbers: [145, 40585] }.');"
+    testString: "assert.deepEqual(digitFactorial(), { sum: 40730, numbers: [145, 40585] });"
```
Comment (moT01) [inline curriculum/challenges/english/08-coding-interview-prep/project-euler/problem-34-digit-factorials.english.md]:
```suggestion
    testString: assert.deepEqual(digitFactorial(), { sum: 40730, numbers: [145, 40585] });
```

**Review by moT01 — COMMENTED** [review]
I was just going to merge this one, but I noticed that just this one has quotes around the text. Not sure if we want to clean that up here, or just leave it. Let me know if you want to leave it and I will merge this @RandellDawson.

File: curriculum/challenges/english/08-coding-interview-prep/project-euler/problem-34-digit-factorials.english.md (hunk 1 above)
Comment (RandellDawson) [reply curriculum/challenges/english/08-coding-interview-prep/project-euler/problem-34-digit-factorials.english.md]: @moT01 Your suggested change shows removing a space before the `- text:` which would result in invalid yaml.  I don't mind removing the surrounding double quotes though.  I will just take care of removing them in a separate commit.

**Comment by RandellDawson** [comment]
@moT01 Just realized after removing the double quotes around that testString that the colons were going to cause a problem with the yaml, so I put them back.  This needs to be merged as is.

**Review by moT01 — APPROVED** [review]
LGTM @RandellDawson 🎉

**Pull request merged**